db/migrate/20221031115646_create_notes.rb

```diff
@@ -0,0 +1,16 @@
+class CreateNotes < ActiveRecord::Migration[6.0]
+  def change
+    create_table :notes do |t|
+      t.boolean :issue, default: false
+      t.string :title, null: false
+      t.text :description, null: false
```

Comment:
I think this should be a rich text field, similar to what we're using for e.g. activity types, etc.